test/test_product_config.py

```diff
@@ -1,6 +1,7 @@
 """Tests for :mod:`katsdpcontroller.product_config`."""
 
 import copy
+import re
```

Comment:
Did black do this? There are some `re.escape()` calls down below, which strikes me as an odd thing for an automated tool to add.

Reply:
Go read the PR description again.